os/hal/ports/SN32/LLD/SN32F2xx/USB/hal_usb_lld.c

```diff
@@ -137,6 +137,11 @@ static void sn32_usb_read_fifo(usbep_t ep, uint8_t *buf, size_t sz, bool intr) {
         if (off + chunk > sz)
             chunk = sz - off;
 
+#if defined(SN32F240)
```

Comment:
having it like
```suggestion
#if (SN32_USB_DIRECT_SRAM == TRUE)
```
would be preferred. Set the behavior true/false on all sn32 boards in the registry